plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -17,24 +19,34 @@ import { PluginTaskScheduler } from '@backstage/backend-tasks';
 import { TaskRunner } from '@backstage/backend-tasks';
 import { UrlReader } from '@backstage/backend-common';
 
+// @public
+export const ANNOTATION_AWS_ACCOUNT_ID: string;
+
+// @public
+export const ANNOTATION_AWS_ARN: string;
+
 // @public
 export type AWSCredentialFactory = (
   awsAccountId: string,
 ) => Promise<AwsCredentialIdentity>;
 
 // @public
 export class AwsEKSClusterProcessor implements CatalogProcessor {
-  constructor(options: {
+  constructor(options?: {
     credentialsFactory?: AWSCredentialFactory;
     credentialsManager?: AwsCredentialsManager;
+    clusterEntityTransformer?: EKSClusterEntityTransformer;
   });
   // (undocumented)
-  static fromConfig(configRoot: Config): AwsEKSClusterProcessor;
+  static fromConfig(
+    configRoot: Config,
+    options?: {
+      clusterEntityTransformer?: EKSClusterEntityTransformer;
+    },
+  ): AwsEKSClusterProcessor;
   // (undocumented)
   getProcessorName(): string;
   // (undocumented)
-  normalizeName(name: string): string;
```

Comment:
This is a breaking change, however, you already use `minor` in the related changeset. ✔️

Reply:
normalizeName was moved to `src/lib/defaultTransformer.ts` as it is part of/being used in the EKS entity default callback function